src/libpcp/src/util.c

```diff
@@ -3162,3 +3166,235 @@ __pmLogFeaturesStr(__uint32_t features)
 
     return(ans);
 }
+
+/* map inode type part of st_mode from lstat() -> string */
+static char *
+inodetype(mode_t st_mode)
+{
+    switch (st_mode & S_IFMT) {
+	case S_IFBLK:	return "block device"; break;
+	case S_IFCHR:	return "character device"; break;
+	case S_IFDIR:	return "directory"; break;
+	case S_IFIFO:	return "FIFO/pipe"; break;
+	case S_IFLNK:	return "symlink"; break;
+	case S_IFREG:	return "regular file"; break;
+	case S_IFSOCK:	return "socket"; break;
+	default:	return "unknown?"; break;
```

Comment:
I think most of the "break;" use here (after return) can be dropped?  Personal style, I'd just keep the default: case as a "break;" and return unknown on the final line of the function.

Reply:
Fixed in commit f29a6af00